cmd/jaeger/internal/extension/jaegerstorage/extension_test.go

```diff
@@ -151,6 +157,43 @@ func TestBadgerStorageExtensionError(t *testing.T) {
 	require.ErrorContains(t, err, "/bad/path")
 }
 
+func TestESStorageExtension(t *testing.T) {
+	mockEsServerResponse := []byte(`
+	{
+		"Version": {
+			"Number": "6"
+		}
+	}
+	`)
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write(mockEsServerResponse)
+	}))
+	defer server.Close()
+	storageExtension := makeStorageExtenion(t, &Config{
+		Elasticsearch: map[string]esCfg.Configuration{
+			"foo": {
+				Servers:  []string{server.URL},
+				LogLevel: "error",
+			},
+		},
+	})
+	ctx := context.Background()
+	err := storageExtension.Start(ctx, componenttest.NewNopHost())
+	require.NoError(t, err)
+	require.NoError(t, storageExtension.Shutdown(ctx))
+}
+
+func TestESStorageExtensionError(t *testing.T) {
+	ext := makeStorageExtenion(t, &Config{
+		Elasticsearch: map[string]esCfg.Configuration{
+			"foo": {},
+		},
+	})
+	err := ext.Start(context.Background(), componenttest.NewNopHost())
+	require.ErrorContains(t, err, "failed to initialize elasticsearch storage")
+	require.ErrorContains(t, err, "no servers specified")
```

Comment:
I think this should be caught during config validation. We need to add annotations to esCfg.Config struct.

Reply:
added annotations